### DISCUSSION TITLE
πρόσθετος χρόνος υποβολής

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#83] [Q&A] [answered]
Καλημέρα σας,

θα ήθελα να ρωτήσω αν η εξέταση θα διαρκέσει 3 ώρες και αν μέσα σε αυτό το διάστημα θα πρέπει να έχουμε υποβάλει το pdf στο exams ή θα μας δοθεί το καθιερωμένο έξτρα 15λεπτο για την υποβολή όπως στις άλλες εξετάσεις.

**Comment by epidrome** [comment, marked as answer]
η εξέταση διαρκεί 3 ώρες, αλλά έχω προσθέσει 50% επιπλέον χρόνο για την κατάθεση και για όποιον θέλει περισσότερο χρόνο για την ανάπτυξη των θεμάτων

**Reply by afoscolou** [reply]
Στο exams ο χρόνος υποβολής είναι 3 ώρες. Λήγει στις 20:30. Θα το αλλάξετε;

**Reply by epidrome** [reply]
λήγει τυπικά έτσι ώστε να εκτιμηθεί και η ταχύτητα-συντομία σας

πρακτικά μπορείτε να καταθέσετε μέχρι τις 10μμ ως εκπρόθεσμη αν κάποιος το έχει πραγματικά ανάγκη για τεχνικούς λόγους κυρίως και λιγότερο για να ολοκληρώσετε το περιεχόμενο, αλλά δεν το συστήνουμε
